build.gradle.kts

```diff
@@ -13,11 +13,6 @@ plugins {
   scala
 }
 
-project.extensions.add(
```

Comment:
WDYT @MaciejG604? I don't think using an extension is justified/necessary here, `intellijVersions` don't provide any methods, just read-only properties

Reply:
Yeah, that is true.